micall/core/aln2counts.py

```diff
@@ -1074,6 +1085,26 @@ def merge_extra_counts(self):
                             report_nuc.seed_nucleotide.insertion_count += \
                                 seed_insertion_counts[query_pos]
                             report_nuc_index += 1
+            if first_amino_index is None and first_nuc_index is not None:
```

Comment:
The previous `if` statement is `if first_amino_index is not None:`, so this can be `elif first_nuc_index is not None:`.

Reply:
Done!